src/main/java/org/jboss/aesh/io/FileResource.java

```diff
@@ -104,6 +105,12 @@ public boolean delete() {
     }
 
     @Override
+    public void move(Resource target) throws IOException {
+        FileResource destination = (FileResource) target;
+        Files.move(this.file.toPath(), destination.file.toPath(), StandardCopyOption.REPLACE_EXISTING);
```

Comment:
I tried in my shell.. the default is always override. The mv only ask if you want override the file if you run it in interactive mode. 


![screenshot from 2015-03-26 06 50 18](https://cloud.githubusercontent.com/assets/350841/6843760/854873da-d384-11e4-81df-d1fff45c3273.png)